WowUp.WPF/Views/AddonsView.xaml

```diff
@@ -267,6 +267,9 @@
                                       Command="{Binding AlphaCheckedCommand}"></MenuItem>
                         </MenuItem>
                         <Separator Visibility="{Binding ShowUpdateButton, Converter={StaticResource BoolToVisibilty}}" />
+                        <MenuItem Header="Open Folder"
```

Comment:
This is fine, maybe move it above the Separator tag so its in the same place regardless of update status.